OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -20,28 +22,37 @@ namespace OpenRA.Mods.Common.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class CaptureProperties : ScriptActorProperties
 	{
-		readonly CapturesInfo normalInfo;
+		readonly Captures[] capturesTraits;
```

Comment:
> Variable captures is used below

This is no longer the case since @atlimit8's suggested change, so you could rename it now. Not a blocker in my eyes, though.